shared/constants/chat2/convostate.tsx

```diff
@@ -1701,9 +1698,26 @@ const createSlice: Z.ImmerStateCreator<ConvoState> = (set, get) => {
           }
           forceContainsLatestCalc = true
         }
-        dispatch.loadMoreMessages({
+
+        // toss pending
```

Comment:
instead of clearing we do need to make sure we don't have anything that conflicts with the incoming data. as far as my small testing is concerned we do get deletes and things like that so we don't need to resolve those issues but we do have pending ordinals which could have been deleted, so we just clear those and adopt the resolved message ids on a clean load